etl/interfaces/interfaces.go

```diff
@@ -81,6 +80,8 @@ type StorageSetter interface {
 // StorageFetcher gets data from database
 type StorageFetcher interface {
 	GetRecord(ref models.Reference) (models.Record, error)
+	GetNotCompletePulses() ([]models.Pulse, error)
```

Comment:
could you please add a description about what does GetNotCompletePulses mean?

Reply:
it might be hard to understand people out of the loop.

Reply:
yes, it's good point! done

Reply:
GetNotCompletePulses -> GetUncompletePulses ?

Reply:
GetIncompletePulses, done